packages/javascript_common/src/app.ts

```diff
@@ -298,7 +298,7 @@ export class App implements AppContext {
         let errMessage = `${err}`;
         if (err instanceof WebAssembly.RuntimeError) {
           errName = 'WebAssemblyRuntimeError';
-          errMessage = err.message;
+          errMessage = err.stack ?? err.message;
```

Comment:
Is this meant to be here or for debugging?

Reply:
I think it's good to have it here. I found out that if `err.stack` is defined then it also contains the message and with the `--debuginfo` wasm-opt flag it now preserves full symbol names in debug WASM blobs - this means we get a stacktrace like this:
```
ERROR: UnexpectedError [WebAssemblyRuntimeError]: RuntimeError: unreachable
    at abort (wasm://wasm/172cc62e:wasm-function[32414]:0xaeb76e)
    at __assert_fail (wasm://wasm/172cc62e:wasm-function[32427]:0xaebd6a)
    at __JS_FreeValueRT (wasm://wasm/172cc62e:wasm-function[13637]:0x4b15d0)
    at JS_CallInternal (wasm://wasm/172cc62e:wasm-function[13635]:0x4a2581)
    at JS_CallInternal (wasm://wasm/172cc62e:wasm-function[13635]:0x4a24bd)
    at JS_CallInternal (wasm://wasm/172cc62e:wasm-function[13635]:0x4a24bd)
    at JS_EvalFunctionInternal (wasm://wasm/172cc62e:wasm-function[13873]:0x4f242f)
    at __JS_EvalInternal (wasm://wasm/172cc62e:wasm-function[13662]:0x4be0e3)
    at JS_Eval (wasm://wasm/172cc62e:wasm-function[13877]:0x4f4077)
    at _ZN15quickjs_wasm_rs10js_binding7context12JSContextRef4eval17h8d5259b8a1155eabE (wasm://wasm/172cc62e:wasm-function[12968]:0x456750)
    at Object.performFn (file:///Users/edward/Documents/Superface/one-sdk/packages/nodejs_host/dist/common/app.js:229:23)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async file:///Users/edward/Documents/Superface/one-sdk/packages/nodejs_host/dist/common/app.js:87:13
    at async AsyncMutex.withLock (file:///Users/edward/Documents/Superface/one-sdk/packages/nodejs_host/dist/common/lib/async_mutex.js:36:20)
    at async InternalClient.perform (file:///Users/edward/Documents/Superface/one-sdk/packages/nodejs_host/dist/index.js:217:20)
    at async UseCase.perform (file:///Users/edward/Documents/Superface/one-sdk/packages/nodejs_host/dist/index.js:303:16)
    at async file:///Users/edward/Documents/Superface/one-sdk/examples/nodejs/index.mjs:31:18
```

for comparison on release mode (and before also on debug) it shows this:
```
ERROR: UnexpectedError [WebAssemblyRuntimeError]: RuntimeError: unreachable
    at wasm://wasm/013652a2:wasm-function[3887]:0x3fb1c1
    at wasm://wasm/013652a2:wasm-function[3224]:0x322e2a
    at wasm://wasm/013652a2:wasm-function[3222]:0x312bc5
    at wasm://wasm/013652a2:wasm-function[3222]:0x312b07
    at wasm://wasm/013652a2:wasm-function[3222]:0x312b07
    at wasm://wasm/013652a2:wasm-function[3416]:0x35d4d7
    at wasm://wasm/013652a2:wasm-function[3243]:0x32c665
    at wasm://wasm/013652a2:wasm-function[1713]:0x17f4b0
    at wasm://wasm/013652a2:wasm-function[1966]:0x1f2f88
    at file:///Users/edward/Documents/Superface/one-sdk/packages/nodejs_host/dist/common/lib/asyncify.js:80:38
    at Object.performFn (file:///Users/edward/Documents/Superface/one-sdk/packages/nodejs_host/dist/common/app.js:229:23)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async file:///Users/edward/Documents/Superface/one-sdk/packages/nodejs_host/dist/common/app.js:87:13
    at async AsyncMutex.withLock (file:///Users/edward/Documents/Superface/one-sdk/packages/nodejs_host/dist/common/lib/async_mutex.js:36:20)
    at async InternalClient.perform (file:///Users/edward/Documents/Superface/one-sdk/packages/nodejs_host/dist/index.js:217:20)
    at async UseCase.perform (file:///Users/edward/Documents/Superface/one-sdk/packages/nodejs_host/dist/index.js:303:16)
    at async file:///Users/edward/Documents/Superface/one-sdk/examples/nodejs/index.mjs:31:18
```